homeassistant/components/group/light.py

```diff
@@ -149,7 +151,8 @@ def should_poll(self) -> bool:
 
     async def async_turn_on(self, **kwargs):
         """Forward the turn_on command to all lights in the light group."""
-        data = {ATTR_ENTITY_ID: self._entity_ids}
+        data = {ATTR_ENTITY_ID: list(self._entity_ids)}
```

Comment:
For clarity, let's not mutate data that is part of `data`, but maintain the list of entity IDs in a standalone var until it's ready.

Reply:
Done.